data/transactions/logic/opcodes.go

```diff
@@ -237,6 +237,7 @@ var OpSpecs = []OpSpec{
 	{0x4e, "cover", opCover, asmDefault, disDefault, oneAny, oneAny, 5, modeAny, stacky(typeCover, "n")},
 	{0x4f, "uncover", opUncover, asmDefault, disDefault, oneAny, oneAny, 5, modeAny, stacky(typeUncover, "n")},
 
+	// byteslice processing / StringOps
```

Comment:
Would have been nice to put this `opCode` in the `Byteslice math` section, but there are no slots available. This seems like a reasonable placement.